### ISSUE TITLE
TODO - Ready to Review

### ISSUE THREAD

**Issue by cbedoy** [cbedoy/TodoApp#1]
Hello guys my application is complete, please review @codepathreview  @codepath

Also I have a question for example if I want to integrate the Suggested features, can I use third party libs like as SugarORM to improving the read and write information or only should be code made for me?

Currently I'm Android Developer but I love develop 

Can I?

Bedoy :)

**Comment by nesquena** [comment]
Feel free to leverage third-party libraries as you are adding extensions

**Comment by nesquena** [comment]
Carlos, are you currently located in the bay area near San Francisco? If not, perhaps it would be better if we provided you a remote self-paced observer access? You won't be able to participate in person or receive code review but can still access the material self-paced and work on the projects. While we find this modality is less effective and requires significantly more internal motivation, would be much easier if you aren't nearby.

**Comment by cbedoy** [comment]
Yes I'm mexican, and currently I live in Mexico, but  if you give me the remote server access that would be amazing.

Thanks

**Comment by nesquena** [comment]
Great, will be in touch with you about next steps soon.
